src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```diff
@@ -137,6 +137,19 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <XdaiConfigFiles Include="configs\*xdai*.cfg" />
+  </ItemGroup>
+
+  <Target Name="AliasingGnosisAndXdaiConfigFiles" BeforeTargets="BeforeBuild">
+    <ItemGroup>
+      <XdaiConfigFiles>
+        <GnosisConfigFile>$([System.String]::Copy('%(Filename)').Replace('xdai', 'gnosis'))</GnosisConfigFile>
```

Comment:
this will change everything so pointed chainspec would be gnosis.json, but we don't have it right?